docs/appendices/release-notes/unreleased.rst

```diff
@@ -209,4 +209,7 @@ Changes
 Fixes
 =====
 
-None
+- Fixed the memory accounting of the circuit breaker for the values which
+  type cannot be defined. Previously, we would account ``0`` bytes for values
+  of the undefined scalar type and ``16`` bytes for values of the undefined
+  arrays types, regardless its size.
```

Comment:
I think this might contain a bit too much implementation details and it doesn't explain how that affected a user. Maybe we could re-phrase this?